apps/wallet-mobile/src/features/Swap/common/navigation.ts

```diff
@@ -13,5 +13,19 @@ export const useNavigateTo = () => {
     selectSellToken: () => navigation.navigate('swap-select-sell-token'),
     startSwap: () => navigation.navigate('swap-start-swap'),
     confirmTx: () => navigation.navigate('swap-confirm-tx'),
+    submittedTx: () => navigation.navigate('swap-submitted-tx'),
+    failedTx: () => navigation.navigate('swap-failed-tx'),
+    swapOpenOrders: () =>
+      navigation.reset({
```

Comment:
it needs the entire path, otherwise it will cut the rest of the navigation state, check the helper -> resetToTxHistory. 
```suggestion
      navigation.reset({
```

Reply:
@banklesss can you check the state? In my head it should be `tx-history` -> then the swap-start, but not sure.